lib/common/hash.go

```diff
@@ -22,6 +22,13 @@ const (
 // Hash used to store a blake2b hash
 type Hash [32]byte
 
+// EmptyHash returns Hash which is an array of 32bytes with
+// every byte set to zero.
+func EmptyHash() Hash {
+	res := Hash{}
+	return res
+}
```

Comment:
Yeah we don't want that, there is no point for it and performance-wise it adds a function call.